users.test.js

```diff
@@ -0,0 +1,124 @@
+import request from 'supertest';
+import app from './app';
+
+describe('signup', () => {
+  test('create user success', (done) => {
+    request(app)
+      .post('/users/signup')
+      .send({
+        email: 'hello',
+        password: 'hello',
+        name: 'hello',
+        phoneNumber: 'hello',
+        roadAddress: 'hello',
+        detailAddress: 'hello',
+        zipCode: 'hello',
+      })
+      .set('Accept', 'application/json')
+      .expect(201, { message: 'USER_CREATED_SUCCESSFULLY' })
+      .end(done);
+  });
+
+  test('create user key error', (done) => {
+    request(app)
+      .post('/users/signup')
+      .send({
+        email: 'hello',
+      })
+      .set('Accept', 'application/json')
+      .expect(400, {
+        message:
+          'KEY_ERROR password,name,phoneNumber,roadAddress,detailAddress,zipCode',
+      })
+      .end(done);
+  });
+
+  test('create user key error', (done) => {
+    request(app)
+      .post('/users/signup')
+      .send({
+        password: 'hello',
+      })
+      .set('Accept', 'application/json')
+      .expect(400, {
+        message:
+          'KEY_ERROR email,name,phoneNumber,roadAddress,detailAddress,zipCode',
+      })
+      .end(done);
+  });
+
+  test('create user key error', (done) => {
+    request(app)
+      .post('/users/signup')
+      .send({
+        email: 'hello',
+        password: 'hello',
+      })
+      .set('Accept', 'application/json')
+      .expect(400, {
+        message: 'KEY_ERROR name,phoneNumber,roadAddress,detailAddress,zipCode',
+      })
+      .end(done);
+  });
+
+  test('create user existing user', (done) => {
+    request(app)
+      .post('/users/signup')
+      .send({
+        email: 'hello',
+        password: 'hello',
+        name: 'hello',
+        phoneNumber: 'hello',
+        roadAddress: 'hello',
+        detailAddress: 'hello',
+        zipCode: 'hello',
+      })
+      .set('Accept', 'application/json')
+      .expect(409, { message: 'ALREADY_EXISTING_USER' })
+      .end(done);
+  });
+});
+
+describe('login', () => {
+  test('use login', (done) => {
+    request(app)
+      .post('/users/login')
+      .send({
+        email: '',
+        password: '',
+      })
+      .set('Accept', 'application/json')
+      .expect(400, {
+        message: 'KEY_ERROR',
+      })
+      .end(done);
+  });
```

Comment:
```suggestion
describe('login', () => {
  test('use login', (done) => {
    request(app)
      .post('/users/login')
      .send({

      })
      .set('Accept', 'application/json')
      .expect(400, {
        message: 'KEY_ERROR',
      })
      .end(done);
  });
```

키에러가 나는 경우는 email에 값이 안들어왔을 때가 아니라,
진짜로 바디에 email이라는 키가 없을 때를 말합니다 :)